packages/lib/utils/joplinCloud.ts

```diff
@@ -409,3 +409,18 @@ export function getPlans(stripeConfig: StripePublicConfig): Record<PlanName, Pla
 		},
 	};
 }
+
+enum AccountType {
+	Default = 0,
+	Basic = 1,
+	Pro = 2,
+	Team = 3,
+}
+
+export function accountTypeToString(accountType: AccountType): string {
+	if (accountType === AccountType.Default) return 'Default';
+	if (accountType === AccountType.Basic) return 'Basic';
+	if (accountType === AccountType.Pro) return 'Pro';
+	if (accountType === AccountType.Team) return 'Team';
+	throw new Error(`Invalid type: ${accountType}`);
```

Comment:
**Optional:** Add a compile-time [exhaustiveness assertion](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#exhaustiveness-checking) here:
```suggestion
	const exhaustivenessCheck: never = accountType;
	throw new Error(`Invalid type: ${exhaustivenessCheck}`);
```
This should cause compilation to fail if a new `AccountType` is added and `accountTypeToString` is not updated.